### PR TITLE
Created an example for 'Get file contents' functionality

### PR DESCRIPTION
**Pull request by b-l-u-e**
**Description**:
This PR adds a get-file-contents example to demonstrate how to retrieve file contents using the Hedera JavaScript SDK. This functionality is important for developers who need to access and work with file data stored on the Hedera network.

* Add get-file-contents.js example file
* Implement file contents query using FileContentsQuery
* Include error handling and logging for better developer experience
* Demonstrate proper use of ECDSA keys or ED25519 keys with the SDK

**Related issue(s)**:

Fixes # 2513

### CODE REVIEW

**Comment by ivaylogarnev-limechain** [comment]
Have you tested this example to ensure it runs correctly?  🔧 

Will this work on a local network setup as well?  👾

**Comment by b-l-u-e** [comment]
Thank you for your guidance.
let me update my code and do further tests

File: examples/get-file-contents.js [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    // Step 1: Client and operator setup
+    const operatorId = AccountId.fromString(process.env.OPERATOR_ID);
+    const operatorKey = PrivateKey.fromStringECDSA(process.env.OPERATOR_KEY);
+    const client = Client.forName(process.env.HEDERA_NETWORK || "testnet");
+    client.setOperator(operatorId, operatorKey);
+
+    console.log("Client and operator setup complete.");
+
+    // For this example, we'll use a known file ID. In practice, you'd use an existing file ID.
+    const fileIdStr = process.env.FILE_ID;
```
[the remark this answers is not in the document]
Comment (b-l-u-e) [reply examples/get-file-contents.js:29]: Alright

**Comment by b-l-u-e** [comment]
am trying to run node get-files-contents.js:

I keep getting an error: Error occurred during file creation: transaction 0.0.1022@1728760988.029461830 failed precheck with status PAYER_ACCOUNT_NOT_FOUND against node account id 0.0.3

File: examples/get-file-contents.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (b-l-u-e) [reply examples/get-file-contents.js]: i made some changes for fileCreateTransaction

**Comment by b-l-u-e** [comment]
The code is tested and run it on local node. updated the file 
its working fine now.

File: examples/get-file-contents.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (b-l-u-e) [reply examples/get-file-contents.js]: this part is sorted. thanks for pointing it out

**Comment by b-l-u-e** [comment]
Hello team,
i need some help in regards to commit change made 13 hours ago
seems i appear on other peoples commit works instead of mine only.
please help me to resolve this

**Comment by ivaylogarnev-limechain** [comment]
Thank you so much for your great work on the project! I just have a small request regarding the PR. Could you please create a new PR by cloning from the main branch and include the new file in a single commit? This will help avoid any issues with references or CI errors due to the existing 32 commits.

Thanks again for your contribution, and I appreciate your understanding!

**Comment by b-l-u-e** [comment]
@ivaylogarnev-limechain thanks for the feedback.
I have a question. since there are 2 issues am working on right now. and based on the suggestion that i create a new PR and use get file contents" does it mean i dont have to create 2 pull requests for two separate issues. i can still maintain one PR as i work on different issues

**Comment by b-l-u-e** [comment]
I created the PR what happens to this now? should i close it or delete it? Please advice

**Comment by agadzhalov** [comment]
Closing due to new [PR](https://github.com/hashgraph/hedera-sdk-js/pull/2579) opened for this issue

**Pull request closed without merging**